students/yujeong/users/views.py

```diff
@@ -1,5 +1,38 @@
 import json
+import re
 
 from unittest     import result
 from django.http  import JsonResponse
 from django.views import View
+from users.models import User
+
+class SignUpView(View):
+       def post(self, request):
+        data = json.loads(request.body) 
+        email_form = re.compile('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
+        password_form = re.compile('^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$')
```

Comment:
1. 정규표현식을 사용할 경우, 자주 사용되는 경우가 아니라면 `compile()`해서 사용하지 않고, `match()`를 사용하는 것을 공식 문서에서 추천하고 있습니다.
2. 정규 표현식 같은 경우 변수라기 보다는 상수입니다. python에서 상수는 대문자로 만듭니다.

```suggestion
        data = json.loads(request.body
        
        REGEX_EMAIL    = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
        REGEX_PASSWORD = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'

        if not re.match(REGEX_EMAIL, data['email'):
            return JsonResponse({'message':'Invalid email format'}, status=400)
         
        if not re.match(REGEX_PASSWORD, data['password'):
            return JsonResponse({'message':'Invalid password format'}, status=400)
```